src/main/java/cubicchunks/server/PlayerCubeMap.java

```diff
@@ -115,18 +115,18 @@
 	 * but these cubes are not fully loaded/generated yet.
 	 * <p>
 	 * Note that this is not the same as toGenerate list.
-	 * Cube can be loaded while not being fully generated yet (not in the last GeneratorPipeline stage).
+	 * Cube can be loaded while not being fully generated yet (not in the last GeneratorStageRegistry stage).
```

Comment:
What do you mean? I did a simple refactor and this replacement seems fine to me.